api/src/main/resources/workbench.yaml

```diff
@@ -4407,6 +4426,28 @@ definitions:
         items:
           type: string
 
+  PublicInstitutionDetails:
+    type: object
+    description: >
+      Represents an institution which has been approved to validate researchers who wish to use the system.
+      Does not include the sensitive email patterns used to validate users.
+    required:
+      - shortName
```

Comment:
Could this contain an institution with minimal details? It feels like there's some overlap which implies we could build a hierarchy?